apps/web/src/pages/tx/[hash].tsx

```diff
@@ -325,28 +277,95 @@ const Tx: NextPage = () => {
   );
 };
 
-type BlockHashProps = {
-  partialHash: string;
-  fullHash: string | undefined;
+type OptimismCardProps = {
+  decodedData: OptimismDecodedData;
+  txTimestamp: dayjs.Dayjs | undefined;
 };
 
-const BlockHash: FC<BlockHashProps> = ({ fullHash, partialHash }) => {
-  if (fullHash === undefined) {
-    return "0x" + partialHash + "...";
-  }
+const OptimismCard: FC<OptimismCardProps> = ({ decodedData, txTimestamp }) => {
```

Comment:
I would extract this component into its own file

Reply:
Done [d3f290e](https://github.com/Blobscan/blobscan/pull/658/commits/d3f290ed19cce89789e670f58976bbabd0817f7f)